sklearn/semi_supervised/tests/test_label_propagation.py

```diff
@@ -12,68 +14,70 @@
 from sklearn.exceptions import ConvergenceWarning
 from numpy.testing import assert_array_almost_equal
 from numpy.testing import assert_array_equal
+from sklearn.utils._testing import _convert_container
 
-ESTIMATORS = [
-    (label_propagation.LabelPropagation, {"kernel": "rbf"}),
-    (label_propagation.LabelPropagation, {"kernel": "knn", "n_neighbors": 2}),
-    (
+CONSTRUCTOR_TYPES = ("array", "sparse_csr", "sparse_csc")
+
+
+def label_estimator():
```

Comment:
How much does this increase the runtime of running this test file? In other words, how long does the following command take on `main` vs this PR?

```bash
pytest sklearn/semi_supervised/tests/test_label_propagation.py
```

Reply:
On my machine, `main` runs in ~0.55s, this pr runs in ~0.60s.